mopidy/config/__init__.py

```diff
@@ -60,6 +60,8 @@
 _audio_schema["output"] = String()
 _audio_schema["visualizer"] = Deprecated()
 _audio_schema["buffer_time"] = Integer(optional=True, minimum=1)
+_audio_schema["min_buffer_size"] = Integer(optional=True, minimum=0)
+_audio_schema["min_buffer_duration"] = Integer(optional=True, minimum=0)
```

Comment:
Given our code doesn't make any attempt to modify these values dynamically, I think it's probably better if the existing convention is followed by removing `min_` from the start of the setting names.

Reply:
No mopidy doesnt modify these values; what is meant is that this buffer is to be filled fully to the value specified, before playing starts. So the larger this value, the larger the latency. The buffer itself might be larger, or not, only GStreamer devs would know, but that is IMHO not relevant here....

Reply:
I agree these names should change.